File: samples/client/petstore/csharp-dotnet2/OpenAPIClientTest/Lib/OpenAPIClient/src/main/CsharpDotNet2/Org/OpenAPITools/Model/InlineObject1.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineObject1 {
    /// <summary>
    /// Additional data to pass to server
    /// </summary>
    /// <value>Additional data to pass to server</value>
    [DataMember(Name="additionalMetadata", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "additionalMetadata")]
    public string AdditionalMetadata { get; set; }

    /// <summary>
    /// file to upload
    /// </summary>
    /// <value>file to upload</value>
    [DataMember(Name="file", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "file")]
    public System.IO.Stream File { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineObject1 {\n");
      sb.Append("  AdditionalMetadata: ").Append(AdditionalMetadata).Append("\n");
      sb.Append("  File: ").Append(File).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: samples/client/petstore/dart/flutter_petstore/openapi/lib/model/inline_object1.dart
part of openapi.api;

class InlineObject1 {
  /* Additional data to pass to server */
  String additionalMetadata = null;
  /* file to upload */
  MultipartFile file = null;
  InlineObject1();

  @override
  String toString() {
    return 'InlineObject1[additionalMetadata=$additionalMetadata, file=$file, ]';
  }

  InlineObject1.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['additionalMetadata'] == null) {
      additionalMetadata = null;
    } else {
      additionalMetadata = json['additionalMetadata'];
    }
    if (json['file'] == null) {
      file = null;
    } else {
      file = new File.fromJson(json['file']);
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'additionalMetadata': additionalMetadata,
      'file': file
    };
  }

  static List<InlineObject1> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineObject1>() : json.map((value) => new InlineObject1.fromJson(value)).toList();
  }

  static Map<String, InlineObject1> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineObject1>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineObject1.fromJson(value));
    }
    return map;
  }
}

